README.md

```diff
@@ -20,16 +20,52 @@ Overview
 
 The goal of this package is to simplify plotting in R. This includes improving the basic workflow and using defaults that tend to adhear to USGS style guidelines. Extra features and functions are included that are not available in base for common USGS elements (error bars, callouts, reverse y axes, etc.). This is intended to make the plotting code more concise and easier to interpret for the sake of reproducibility.
 
-| Feature                               | Description                                                                                                            |
-|---------------------------------------|------------------------------------------------------------------------------------------------------------------------|
-| [Piping and plots as objects](#pipes) | Easily add features to your plot.                                                                                      |
-| [Automatic legend](#legend)           | For each plot feature, a corresponding legend name can be specified. Eliminates the need to duplicate `par` arguments. |
-| [Automatic limits](#limits)           | Automatically regenerate limits of plot when new features are added.                                                   |
-| [Error bars](#error)                  | Apply error bars to points by specifying the upper and lower offsets.                                                  |
-| [Callouts](#callouts)                 | Add a line and label in one call.                                                                                      |
-| [Axes reversal](#reverse)             | Reverse the axis by specifying one argument to `axis()`.                                                               |
-| [Embedded functions](#embed)          | Add extra plot features within a points call (no duplication of x/y values)                                            |
-| [Compatibility with base](#base)      | Can start a plot using `gsplot`, and add base R features afterwards.                                                   |
+<table style="width:57%;">
```

Comment:
I think it is a knitr version difference, when I built this the other day it switched to markdown.